crates/nu-command/src/network/http/options.rs

```diff
@@ -97,18 +97,8 @@ impl Command for SubCommand {
                 result: None,
             },
             Example {
-                description: "Get options from example.com, with username and password",
-                example: "http options -u myuser -p mypass https://www.example.com",
-                result: None,
-            },
-            Example {
-                description: "Get options from example.com, with custom header",
-                example: "http options -H [my-header-key my-header-value] https://www.example.com",
-                result: None,
-            },
-            Example {
-                description: "Get options from example.com, with custom headers",
-                example: "http options -H [my-header-key-A my-header-value-A my-header-key-B my-header-value-B] https://www.example.com",
+                description: "Simulate a browser cross-origin preflight request from www.example.com to media.example.com",
+                example: "http options https://media.example.com/api/ -H [Origin https://www.example.com Access-Control-Request-Headers \"Content-Type, X-Custom-Header\" Access-Control-Request-Method GET]",
```

Comment:
Is this actually supposed to return something? It doesn't for me.

Reply:
No, media.example.com is not a real endpoint.

Reply:
ok, thanks for the info.